go/oasis-test-runner/scenario/e2e/runtime/governance_change_parameters.go

```diff
@@ -0,0 +1,136 @@
+package runtime
+
+import (
+	"fmt"
+
+	"github.com/oasisprotocol/oasis-core/go/common/cbor"
+	consensus "github.com/oasisprotocol/oasis-core/go/consensus/api"
+	governance "github.com/oasisprotocol/oasis-core/go/governance/api"
+	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/env"
+	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/log"
+	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/oasis"
+	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/scenario"
+	"github.com/oasisprotocol/oasis-core/go/oasis-test-runner/scenario/e2e"
+	staking "github.com/oasisprotocol/oasis-core/go/staking/api"
+)
+
+// GovernanceConsensusChangeParameters is the governance consensus change parameters scenario.
+var GovernanceConsensusChangeParameters scenario.Scenario = newGovernanceConsensusChangeParametersImpl("governance-change-parameters")
+
+type governanceConsensusChangeParametersImpl struct {
+	governanceImpl
+}
+
+func newGovernanceConsensusChangeParametersImpl(name string) scenario.Scenario {
+	sc := &governanceConsensusChangeParametersImpl{
+		governanceImpl: *newGovernanceImpl(name),
+	}
+	return sc
+}
+
+func (sc *governanceConsensusChangeParametersImpl) Fixture() (*oasis.NetworkFixture, error) {
+	f, err := sc.governanceImpl.Fixture()
+	if err != nil {
+		return nil, err
+	}
+
+	f.Network.DefaultLogWatcherHandlerFactories = append(
+		f.Network.DefaultLogWatcherHandlerFactories,
+		log.AssertJSONContains(
+			"msg",
+			"change parameters proposals are disabled",
+			"change parameters proposals should be disabled",
+		),
```

Comment:
If this needs to be verified I would rather add an explicit log event instead of relying on messages having a certain format (e.g. the same as is done for other similar log events).

Reply:
Removed e2e test so this is not relevant anymore. Will remember for the next time.